POST_HERE/templates/base.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <!-- Required meta tags -->
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+        <title> Post Here </title>
+
+        <link rel="stylesheet" href="https://unpkg.com/picnic">
+    </head>
+
+    <body>
+    {% block content %}
+        <p><a href="/subreddit">Click here</a> to get a JSON file with random subreddit data!</p>
```

Comment:
This will not be needed, we can link directly to the json data on the app without needing to route through the root. Or maybe even making the route.